src/Symfony/Component/HttpFoundation/Tests/RequestTest.php

```diff
@@ -1564,7 +1576,10 @@ private function getRequestInstanceForClientIpsForwardedTests($remoteAddr, $http
         return $request;
     }
 
-    public function testTrustedProxies()
+    /**
+     * @group legacy
+     */
+    public function testLegacyTrustedProxies()
```

Comment:
There is not need to rename the test with the prefix `testLegacy` and add the test to the `legacy` group, with one of both things is enough.